### DISCUSSION TITLE
Support for "file" / "media" field in product editor

### DISCUSSION THREAD

**Discussion by leewillis77** [woocommerce/woocommerce#47528] [WooCommerce New Product Editor]
The new product editor should support a "file" / "media" field type that allows a user to either select an existing media item, or upload a new one. There are many cases where a plugin may want to extend the product editor to allow users to add/select files on a product, for example

- a product "datasheet" to be attached to a product
- a product user-guide / manual
- a product safety certificate / rating document

**Comment by mattsherman** [comment]
Thanks @leewillis77 -- I've created an issue for this: #47821

Closing this discussion -- we'll use the issue to track this and refine requirements.
